src/gui/editorwidgets/qgsrelationreferencewidget.h

```diff
@@ -183,6 +183,7 @@ class GUI_EXPORT QgsRelationReferenceWidget : public QWidget
     void filterChanged();
     void addEntry();
     void updateAddEntryButton();
+    void updateIndex();
```

Comment:
Can you add a short description to this method (even though it's not in the API docs)? It's not trivial to understand what it does (at least for me).